mim/utils/utils.py

```diff
@@ -157,19 +161,23 @@ def get_content_from_url(url: str,
 @typing.no_type_check
 def download_from_file(url: str,
                        dest_path: str,
+                       check_certificate: bool,
```

Comment:
Adding a new parameter in the middle position may cause a bc issue. Suggest moving it to the end.

Reply:
I resolved it.